test/support/clusterpools.go

```diff
@@ -0,0 +1,115 @@
+package support
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	ocmsdk "github.com/openshift-online/ocm-sdk-go"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	mapiclientset "github.com/openshift/client-go/machine/clientset/versioned"
+	"github.com/openshift/client-go/machine/listers/machine/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+var (
+	ClusterID     string = os.Getenv("CLUSTERID")
+	machineClient mapiclientset.Interface
+	msLister      v1beta1.MachineSetLister
+	TestName      string = "test-instascale"
+)
+
+const (
+	namespaceToList = "openshift-machine-api"
+)
+
+func CreateOCMConnection(secret string) (*ocmsdk.Connection, error) {
+	logger, err := ocmsdk.NewGoLoggerBuilder().
+		Debug(false).
+		Build()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't build logger: %v\n", err)
+		os.Exit(1)
+	}
+	connection, err := ocmsdk.NewConnectionBuilder().
+		Logger(logger).
+		Tokens(string(secret)).
+		Build()
+	if err != nil || connection == nil {
+		fmt.Fprintf(os.Stderr, "Can't build connection: %v\n", err)
+		os.Exit(1)
+	}
+
+	return connection, nil
+}
+
+func MachinePoolsExist(connection *ocmsdk.Connection) (bool, error) {
+	machinePools := connection.ClustersMgmt().V1().Clusters().Cluster(ClusterID).MachinePools()
+	return machinePools != nil, nil
+}
+
+func NodePoolsExist(connection *ocmsdk.Connection) (bool, error) {
+	nodePools := connection.ClustersMgmt().V1().Clusters().Cluster(ClusterID).NodePools()
+	return nodePools != nil, nil
+}
```

Comment:
It seems like these functions are not used, can they be deleted?

Reply:
They are no longer needed, I have removed them.